geometry_msgs/PoseStamped[] qrcode_poses